templates/cassiopeia/scss/offline.scss

```diff
@@ -4,20 +4,11 @@
 // Variables, Functions and Mixins
 @import "tools/tools";
 
-html {
-  background: radial-gradient(circle, $gray-100, $gray-200);
-  background-color: $cyan;
-  background-image: none;
-  background-repeat: no-repeat;
-  background-attachment: fixed;
-}
-
 body {
```

Comment:
this entire block is obsolete since template.css is loaded and font-styling is defined over there. 
Am I correct?

Reply:
Yes thats correct template.css and then offline.css are loaded

Reply:
Yes, that is why I removed it

Reply:
sorry.. I meant the body block. Github added the 4 lines above too.

Reply:
Ah, I see... yes, if template.css is also being loaded, the code is not needed here